pyspecProcScripts/correlation_alignment.py

```diff
@@ -77,64 +79,29 @@ def correl_align(
                 the data in the calculation of the correlation function.
     """
     logging.debug("Applying the correlation routine")
-    # TODO ☐: I think we want to rename and throw
-    #         errors for the kwargs "indirect_dim"
-    #         and "avg_dim", and rather use kwargs
-    #         that clearly specify which indirect
-    #         dimensions are *used for alignment*
-    #         (let's call this "repeated") and
-    #         which are not.
-    #         For example, in an inversion
-    #         recovery or E(p) we typically want
-    #         to align along the indirect
-    #         dimension (power or vd is the
-    #         `repeat_dim`), but only AFTER we have
-    #         made it safe to do so by flipping
-    #         the signs.
-    #         As another example, we almost
-    #         *always* want to align along nScans
-    #         (so we would smoosh the indirect
-    #         dimension and `nScans` to make
-    #         `repeat_dim`).
-    #         Most likely, we want to specify a
-    #         list of dimensions that are "safe"
-    #         to smoosh together as `repeat_dim`,
-    #         assuming that `nScans` is *always*
-    #         safe.
-    #         I think that right now, what you
-    #         call `indirect_dim` will be part of
-    #         `repeat_dims`.
-    # TODO ☐: Furthermore, we want to specify
-    #         `non_repeat_dims`.  These are also
-    #         indirect dimension, but they are
-    #         one that we are not OK to align
-    #         (e.g. the indirect dimension of a 2D
-    #         COSY experiment).
-    #         With both `repeat_dims` and
-    #         `non_repeat_dims` defined, we can use a set
-    #         expression like below to find the
-    #         indirect dimension (not phase
-    #         cycling and not direct) and to make
-    #         sure that they are all either (1)
-    #         nScans (2) specified as safe or (3)
-    #         listed in `non_repeat_dims`
-    #         Currently, I don't think you have
-    #         any `non_repeat_dims`
-    if avg_dim:
-        phcycdims = [j for j in s_orig.dimlabels if j.startswith("ph")]
-        indirect = set(s_orig.dimlabels) - set(phcycdims) - set([direct])
-        # TODO ☐: see todo above -- you need to
-        #         first make repeat_dims and
-        #         non_repeat_dims lists of length
-        #         1 if they are given as strings
-        # assert len(indirect - set(repeat_dims) - set(non_repeat_dims)) = 0
-        indirect = [j for j in s_orig.dimlabels if j in indirect]
-        s_jk = s_orig.C.smoosh(indirect)  # this version ends up with
+    assert indirect_dim is None, (
+        "We updated the correlation function to no longer take indirect_dim as"
```

Comment:
I *think* that `indirect_dim` roughly corresponds to `repeated_dims`, so you should say that

Reply:
check